config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
+  namespace :api do
+    get 'greetings/random', to: 'greetings#random'
+  end
+  # get 'greetings/random_greeting', to: 'greetings#random_greeting'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
-end
+  root 'root#index'
```

Comment:
- [ ] Kindly make sure that the root is running the react app instead of the API. The API should only be running on the `api/greetings/random` path